Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Cursors/BaseCursor.cs

```diff
@@ -333,6 +363,7 @@ public virtual void OnPointerUp(MixedRealityPointerEventData eventData)
         protected virtual void Start()
         {
             RegisterManagers();
+            CalculateBounds();
```

Comment:
Agreed, should've thought of that. Will make relevant changes.

Reply:
After further thinking/discussion with others: 

My current feeling is that the cursor bounds being calculated in Start() should establish a baseline rather than a physical measurement. This baseline should remain in perpetuity and not upon cursor visuals (mesh) being updated at runtime. For example, the default reticle has many meshes involved with its lifecycle. These meshes are shown and hidden during various interaction state changes. Let's say a user wants to use a mesh significantly larger than the default resting cursor and swaps it out at runtime. If they incorporate that new mesh into the lifecycle of the cursor, and new bounds are established, that would make all of the other meshes substantially smaller. A user would need to replace every mesh involved with every state of cursor interaction simultaneously for that to be useful. 

I think we should think of various cursors' scaling in relation to the default resting cursor. If a user wishes to make a separate cursor prefab outside of runtime then that is totally possible and the other states will continue to be scaled based on the mesh in the default resting state. Let me know what you think!

Reply:
makes sense, thanks for looking into it!